### ISSUE TITLE
The console output is missing a few line breaks / new lines

### ISSUE THREAD

**Issue by CJDennis** [infection/infection#798]
| Question    | Answer
| ------------| ---------------
| Infection version | 0.14.1
| Test Framework version | PHPUnit 8.4-g5dbaa128f
| PHP version | 7.3.9
| Platform    | Windows 10
| Github Repo | -

The console output is missing a few line breaks / new lines.

There should be a new line before:
- `Running initial test suite...`
- `Processing source code files: 0/3`
- `Creating mutated files and processes:  0/84`
 
 ```
You are running Infection with Xdebug enabled.
     ____      ____          __  _
    /  _/___  / __/__  _____/ /_(_)___  ____
    / // __ \/ /_/ _ \/ ___/ __/ / __ \/ __ \
  _/ // / / / __/  __/ /__/ /_/ / /_/ / / / /
 /___/_/ /_/_/  \___/\___/\__/_/\____/_/ /_/

    0 [>---------------------------] < 1 secRunning initial test suite...

PHPUnit version: 8.4-g5dbaa128f

    4 [============================]  1 secProcessing source code files: 0/3

Generate mutants...

Processing source code files: 3/3Creating mutated files and processes:  0/84
Creating mutated files and processes: 84/84
.: killed, M: escaped, S: uncovered, E: fatal error, T: timed out

........M........M...M....MMMM....M.........M.M...   (50 / 84)
..M..SSSSSSSSSSSSSSSSSSSSSS.MSMMMS                   (84 / 84)

84 mutations were generated:
      45 mutants were killed
      24 mutants were not covered by tests
      15 covered mutants were not detected
       0 errors were encountered
       0 time outs were encountered

Metrics:
         Mutation Score Indicator (MSI): 53%
         Mutation Code Coverage: 71%
         Covered Code MSI: 75%

Please note that some mutants will inevitably be harmless (i.e. false positives).
Dashboard report has not been sent: it is not a Travis CI

Time: 18s. Memory: 16.00MB
 ```

**Comment by maks-rafalko** [comment]
Fixed in https://github.com/infection/infection/pull/799 and will be released in 0.14.2

**Comment by CJDennis** [comment]
@maks-rafalko I've updated to 0.14.2 and only the first line is fixed:

```
You are running Infection with Xdebug enabled.
     ____      ____          __  _
    /  _/___  / __/__  _____/ /_(_)___  ____
    / // __ \/ /_/ _ \/ ___/ __/ / __ \/ __ \
  _/ // / / / __/  __/ /__/ /_/ / /_/ / / / /
 /___/_/ /_/_/  \___/\___/\__/_/\____/_/ /_/

    0 [>---------------------------] < 1 sec
Running initial test suite...

PHPUnit version: 8.3.5

    3 [============================]  1 secProcessing source code files: 0/1

Generate mutants...

Processing source code files: 1/1Creating mutated files and processes:   0/156
Creating mutated files and processes: 156/156
.: killed, M: escaped, S: uncovered, E: fatal error, T: timed out

..................................................   ( 50 / 156)
..................................................   (100 / 156)
..................................................   (150 / 156)
......                                               (156 / 156)

156 mutations were generated:
     156 mutants were killed
       0 mutants were not covered by tests
       0 covered mutants were not detected
       0 errors were encountered
       0 time outs were encountered

Metrics:
         Mutation Score Indicator (MSI): 100%
         Mutation Code Coverage: 100%
         Covered Code MSI: 100%

Please note that some mutants will inevitably be harmless (i.e. false positives).
Dashboard report has not been sent: it is not a Travis CI

Time: 10s. Memory: 18.00MB
```
